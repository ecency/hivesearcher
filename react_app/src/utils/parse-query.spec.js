/* eslint-disable */
import parseQuery from './parse-query';

describe('Parse Query', () => {
    it('(1) Empty string', () => {
        let input = '';
        expect(parseQuery(input)).toMatchSnapshot();
    });

    it('(2) With single parameter ', () => {
        let input = '?q=foo';
        expect(parseQuery(input)).toMatchSnapshot();
    });

    it('(3) With multiple parameter ', () => {
        let input = '?q=foo&sort=best';
        expect(parseQuery(input)).toMatchSnapshot();
    });
});