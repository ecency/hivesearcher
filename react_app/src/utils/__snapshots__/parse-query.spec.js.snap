// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parse Query (1) Empty string 1`] = `Object {}`;

exports[`Parse Query (2) With single parameter  1`] = `
Object {
  "q": "foo",
}
`;

exports[`Parse Query (3) With multiple parameter  1`] = `
Object {
  "q": "foo",
  "sort": "best",
}
`;
